myhpi/core/models.py

```diff
@@ -31,6 +33,10 @@ class InformationPage(Page):
         "RootPage",
     ]
 
+    @property
+    def last_edited_by(self):
+        return self.get_latest_revision().user
```

Comment:
@frcroth Where did you find this function? Even now I simply cannot find it in the [reference](https://docs.wagtail.org/en/stable/reference/pages/model_reference.html#page)

Reply:
https://github.com/wagtail/wagtail/blob/d641f6a9ac4e79b5451d4c366a2f95ab78be1769/wagtail/models/__init__.py#L1004